content/blog/multi-stage-yaml-pipeline-to-create-and-publish-nuget-packages.md

```diff
@@ -0,0 +1,263 @@
+---
+title: "Multi-stage YAML pipeline to create and publish NuGet packages"
+date: 2019-08-26T00:00:00+01:00
+publishdate: 2019-08-26T00:00:00+01:00
+lastmod: 2019-08-26T00:00:00+01:00
+image: "images/multi-stage-yaml-pipeline-to-create-and-publish-nuget-packages/multi-stage-yaml-pipeline-to-create-and-publish-nuget-packages.jpg"
+tags: [ "Azure Pipelines", "Azure DevOps", "NuGet", "Continuous Integration", "YAML" ]
+comments: false
+draft: true
+---
+
+I've recently created a NuGet package that I published on nuget.org. To make this process as smoothed and simple as possible I've created a multi-stage yaml pipeline in Azure DevOps to create and publish my package.
```

Comment:
De zin "To make this process as smoothed and simple as possible" loopt niet helemaal lekker. Pas aan het einde van de zin weet ik dat het process het creëren en publiceren van je package is. Misschien al eerder noemen? In order to make the process of creating and publishing this package, I've created a multi-stage YAML pipeline in Azure DevOps?

Reply:
Dat is inderdaad beter.